packages/core/src/ceramic.ts

```diff
@@ -377,7 +377,7 @@ export class Ceramic implements CeramicApi {
     let anchorService = null
     if (!config.gateway) {
       const anchorServiceUrl =
-        config.anchorServiceUrl || DEFAULT_ANCHOR_SERVICE_URLS[networkOptions.name]
+        config.anchorServiceUrl.replace(/\/$/, '') || DEFAULT_ANCHOR_SERVICE_URLS[networkOptions.name]
```

Comment:
can you add a comment what the replace call is doing?  I find regexes incredibly opaque and difficult to read

Reply:
Would be even better to store the regex in a variable with a helpful name. So no need for comment

Reply:
@stbrody @v-stickykeys Extracted variable.